gbsplay.c

```diff
@@ -221,6 +189,40 @@ static void printinfo()
 	redraw = false;
 }
 
+static void handleuserinput(struct gbs *gbs)
```

Comment:
Please avoid moving functions around, thanks!

Reply:
Sorry; `handleuserinput` newly referred to the redraw functions in order to make the "paused" flag display.  It seemed cleaner to move `handleuserinput` than to add forward declarations for the redraw functions.  Would you prefer the latter over the former?

Reply:
I think it's better to add a forward declaration in this case (so the diff in the function itself more obvious).
It could still be optionally be followed up by a separate commit that only moves the function and removes the forward declaration again.

That said, I'm fine with moving it directly as well, I just missed that it was because of the new call to printstatus.